stackslib/src/net/api/getinfo.rs

```diff
@@ -81,6 +82,7 @@ pub struct RPCPeerInfoData {
     pub genesis_chainstate_hash: Sha256Sum,
     pub unanchored_tip: Option<StacksBlockId>,
     pub unanchored_seq: Option<u16>,
+    pub tenure_height: Option<u64>,
```

Comment:
Adding `default` and `skip_serializing_if` tags should make the deserializer more backwards-compatible. (i.e., if a signer happens to query the RPC info from an older node, the deserializer will interpret the lack of the field as a `None`).

```suggestion
    #[serde(default)]
    #[serde(skip_serializing_if = "Option::is_none")]
    pub tenure_height: Option<u64>,
```